tests/functional/helmholtz/kh_2d.py

```
@@ -0,0 +1,238 @@
+        strict=True,
+    )
+
+
+def config(sim=None, timestamps=None, seed=12334):
+    if sim is None:
+        sim = default_setup()
+    if timestamps is None:
+        timestamps = default_timestamps()
+
+    def density(x, y):
+        L = sim.simulation_domain()[1]
```

Ruff (0.12.2)

Local variable `L` is assigned to but never used

Remove assignment to unused variable `L`

(F841)